.github/workflows/ci.yml

```diff
@@ -61,6 +61,20 @@ jobs:
           prometheus_password: ${{ secrets.PROMETHEUS_PASSWORD || '' }}
           loki_username: ${{ secrets.LOKI_ID || '' }}
           loki_password: ${{ secrets.LOKI_PASSWORD || '' }}
+  c-chain-reexecution:
```

Comment:
Alternative to modifying `run-monitored-tmpnet-cmd` to support this case is to copy the required setup here. Leaving this for now.